package.json

```
@@ -4,8 +4,8 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "dev": "concurrently \"loam dev --build-clients\" \"vite\"",
-    "start": "concurrently \"loam dev --build-clients\" \"vite\"",
+    "dev": "concurrently \"stellar-scaffold dev --build-clients\" \"vite\"",
+    "start": "concurrently \"stellar-scaffold dev --build-clients\" \"vite\"",
```

Let's use it as a plugin to `stellar-cli`, because it is

```suggestion
    "dev": "concurrently \"stellar scaffold dev --build-clients\" \"vite\"",
    "start": "concurrently \"stellar scaffold dev --build-clients\" \"vite\"",
```